webboot/webboot.go

```diff
@@ -8,9 +8,42 @@
 package main
 
 import (
-	"fmt"
+	"flag"
+	"log"
+	"os"
+
+	"github.com/u-root/webboot/pkg/mountkexec"
+)
+
+type distro struct {
+	kernel       string
+	initrd       string
+	cmdline      string
+	downloadLink string
+}
+
+var (
+	cmd       = flag.String("cmd", "", "Command Line")
+	mountDir  = flag.String("dir", "mountDir", "The mount point of the ISO")
+	osSystems = map[string]distro{
+		"TinyCore": distro{"/boot/vmlinuz64 ", "/boot/initrd", "--reuse-cmdline", "https:louis.com"},
+	}
 )
 
 func main() {
-	fmt.Println("weboot up")
+
+	flag.Parse()
+	if len(flag.Args()) < 1 {
+		log.Fatal("error:pass in a valid operating system")
+		os.Exit(1)
+	}
+
+	osname := flag.Args()[0]
+	mountPath := *mountDir
+
+	if err := mountkexec.MountISO(osname, mountPath); err != nil {
+		log.Fatalf("error in mountISO:%v", err)
+		os.Exit(1)
```

Comment:
don 't need this os.Exit()

Reply:
Yes I agree also since log.Fatalf exists also.